mail/views.py

```diff
@@ -237,3 +238,37 @@ def post(self, request, *args, **kwargs):
             status=mailgun_response.status_code,
             data=generate_mailgun_response_json(mailgun_response)
         )
+
+
+class EmailBouncedView(APIView):
+    """
+    View class that handles HTTP requests to course team mail API
+    """
+    permission_classes = (permissions.AllowAny, )
```

Comment:
Please secure this endpoint. There is an HMAC signature which is used to verify authenticity.

Reply:
Done